lib/prawn/embedded_files/filespec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Prawn
+  module EmbeddedFiles
+    class Filespec
+      attr_reader :file_name, :not_hidden
+
+      def initialize(file, options = {})
+        name = options[:name] || file.chksum
+
+        @file_name = PDF::Core::LiteralString.new(name)
+
+        if options[:description]
+          desc_str = options[:description].to_s
+          @description = PDF::Core::LiteralString.new(desc_str)
+        end
+
+        if options[:hidden].nil? || options[:hidden] != true
+          @not_hidden = true
+        end
```

Comment:
Can't you just say `@not_hidden = !@options[:hidden]`?

Also: The only reference for this is on line 28 where the negation is wanted. Why not just store `@hidden` directly and rename the method to `#hidden?`? Then one would not need to mentally do "not not hidden".

Reply:
Yes and no, I was overthinking about this a little bit. I have made some changes and left a comment below with a brief explanation